portality/templates/formcontext/maned_journal_review.html

```diff
@@ -95,8 +95,9 @@ <h3 class="heading-aligned-with-fields">Old journal information</h3>
 
         </div>
 
-        {# include the base journal form which contains all the biblio data #}
-        {% include 'formcontext/_base_journal_form.html' %}
+        {# include the base form which contains all the biblio data #}
+        {# we use the base application here because it contains extra information regarding the DOAJ seal #}
```

Comment:
OK I double checked and at the moment the base journal form does not contain anything the base application form doesn't have as well. That could change however.

Also feels weird to include the base app form in the ManEd *journal* review form.

To tackle both of these "smells", can we refactor the seal to be included in a file separate from the base application file? Just thinking about maintainability here, the journal and application files are driven by quite different needs and changes to either may occur without consulting the other. Making the ManEd journal form **not** based on the base journal form feels quite wrong.

Currently we have ```formcontext/_basic_biblio_fields.html```. How about a ```formcontext/_seal.html``` which is then included in ```formcontext/_base_application_form.html``` as well as ```formcontext/maned_journal_review.html```? Just using an include, the same way basic_biblio_fields is included in both files already.

In other words, instead of using the file which already has what we want, we refactor the seal content out of that file into a separate include, and then very explicitly include the seal content where we want it. The explicitness (yes, it's a word) and the composition together make it clear what is (semantically) supposed to be on the screen and we keep our "basic journal form content is shared by all journals forms" hierarchy in place.